supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,66 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Benefits of internal linking
+* Users first: Allows users to navigate intuitively.
+* Allows Google to understand architecture of the site.
+* Establishes a quantifiable pathway to recognize topical expertise.
+* They increase site visitation time.
+* Helps Google crawl your site.
+* Builds relevancy and ranking signals for keywords and topics.
+* Builds citation or helps users add context.
+
+.Technical best practices
+* Use standard hyperlinks:
+`<a href="http://www.same-domain.com/" title="Keyword Text">Keyword Text</a>`
+* Do not use non hyperlinks.
+* Do not use JavaScript links. 
+* Do not use Flash Links.
+* Do not use links in Frames or iframes
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Search engines have a soft-cap of crawling around 150 links. Avoid over-linking and consider that the navigational links are crawlable too. 
+
+.Anchor text best practices
+* Contextually describe what the user will find post-click.
+* Contextually describe the content at the link location so that the user can decide if they want to leave their current location. 
+* Avoid irrelevant anchor text.
+* You can use a sentence or sentence fragment as the link anchor text.
+
+
 [[internal-links]]
-== Internal links
+== Example format for internal links
```

Comment:
Just so we're all on the same page, what are we saying that the meaning of internal links are vs. external links?

Reply:
I think we are not all on the same page with that. We should consult the experts maybe.

The examples I added are mainly based on this assumption:

- Internal = within the same 'document'. Usually an xref with link text. The reader does not 'leave' their current location.

- External = outside the current 'document' (or user story or whatever). Usually as a hyperlink with link text. The reader leaves their current location. This is why context is so important to include in this instance. To let the reader know where they are being sent by the link so they can decide if they want to 'leave' their current location.

@stoobie's comment about third-party links makes me wonder if we should include such guidance:

"If linking to third party sites, then including a URL instead of a hyperlink is a good way to avoid broken links. In fact, it's best to provide a high-level URL with a non-link title (e.g. For more info, see Installing a vGPU at www.nvidia.com). At my previous workplace, this was our stated guidance (not to use hyperlinks to third-party sites) True, when the link works it's less useful, but when the link doesn't work, it's a lot more useful and less annoying."

But IBM's guidance seems to suggest not to do this:
"Referring to web addresses
In online information, when you provide a link to a web page, use a meaningful descriptive label for the link, not the web address itself. In online information, the web address can be easily identified by hovering over the label. In the label, try to include text that mirrors the title of the linked site, so that users are confident that they are at the correct site after they click the link. Also, try to include keywords in the label; the inclusion of keywords improves search results."

Thoughts?

Reply:
I think that if we have a technical solution in place that is part of CCS's automatic tooling, then we can go ahead with IBM's guidance. If not (which is the case right now), then following this guidance is a recipe for adding broken links to our documentation. Maybe not today, maybe not tomorrow, but soon.